code/controllers/subsystem/air.dm

```diff
@@ -521,7 +521,7 @@ SUBSYSTEM_DEF(air)
 /datum/controller/subsystem/air/proc/setup_allturfs()
 	var/list/turfs_to_init = block(locate(1, 1, 1), locate(world.maxx, world.maxy, world.maxz))
 	var/list/active_turfs = src.active_turfs
-	var/times_fired = ++src.times_fired
```

Comment:
muh cache

Reply:
it's only used once, since I replaced it with time.
tho now that I mention it. I need to figure out how to ensure time doesn't block valid share attempts